activation/poet_test.go

```diff
@@ -62,14 +59,61 @@ func TestHTTPPoet(t *testing.T) {
 	signature := signer.Sign(signing.POET, ch.Bytes())
 	prefix := bytes.Join([][]byte{signer.Prefix(), {byte(signing.POET)}}, nil)
 
-	poetRound, err := client.Submit(context.Background(), prefix, ch.Bytes(), signature, signer.NodeID(), PoetPoW{
+	poetRound, err := client.Submit(context.Background(), time.Time{}, prefix, ch.Bytes(), signature, signer.NodeID(), PoetPoW{
```

Comment:
I'm surprised this works. Is the `HTTPPoetTestHarness` not configured correctly or an older version that is not yet aware about deadlines?

Or does the zero value of `time.Time` mean no deadline?

Reply:
The zero value means no deadline: https://github.com/spacemeshos/poet/blob/bc84a36c1c4f9e7b16bbb230bccdefa7d1c94a71/registration/registration.go#L306